packages/ia-topnav/src/styles/nav-search.js

```diff
@@ -0,0 +1,134 @@
+import { css } from 'lit-element';
+
+export default css`
+  input:focus {
+    outline: none;
+  }
+  button {
+    background: none;
+    color: inherit;
+    border: none;
+    font: inherit;
+    cursor: pointer;
+  }
+  button:focus {
+    outline: none;
+  }
+  .search {
+    padding-top: 0;
+    margin-right: .5rem;
+  }
+  .search svg {
+    position: relative;
+    top: -5px;
+    right: -3px;
+    fill: var(--activeSearchColor);
+  }
+  .search-activated {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    position: absolute;
+    top: 0;
+    right: 5rem;
+    bottom: 0;
+    left: 5rem;
+    z-index: 3;
+    padding: 0.5rem 0.2rem;
+    border-radius: 1rem 1rem 0 0;
+    background: var(--grey20);
+  }
+  .search-inactive {
+    display: none;
+  }
+  .search-activated .highlight,
+  .search-activated .search {
+    background: var(--white);
+    border-radius: 0.5rem;
+  }
+  .search-activated .highlight {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    width: 100%;
+    margin: 0 .5rem;
+  }
+  .search-activated .search {
+    height: 100%;
+    padding: 0;
+    margin-right: 0;
+    -ms-flex-item-align: center;
+    -ms-grid-row-align: center;
+    align-self: center;
+  }
+  .search-activated .search-field {
+    width: 100%;
+    height: 100%;
+    box-sizing: border-box;
+    padding-left: 1rem;
+    border-radius: 0.5rem;
+    border: none;
+    font-size: 1.6rem;
+    text-align: center;
+  }
+  .search-activated .search-field:focus {
+    outline: none;
+  }
+  @keyframes fade-in {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  .fade-in {
+    animation: fade-in .2s forwards;
+  }
+
+  @media (min-width: 890px) {
+    :host {
+      float: right;
```

Comment:
Does this cause layout changes in the parent? If so, I would move the responsibility up a level.